img/css-keyframes-spinner.svg

```diff
@@ -0,0 +1,27 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="150" height="150" viewBox="-75 -75 150 150">
```

Comment:
It would be nice if you could remove the `width` and `height` attributes so that the file will scale smoothly.

Reply:
Ah, good point. I always recommend `width` and `height` for reasonable default sizes, but for the gallery, you want them to be full size!

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="-75 -75 150 150">
```